scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -1170,7 +1172,7 @@ public Void doInTransaction(Session session) {
                     }
                     TaskData taskData = saveNewTask(session, jobRuntimeData, task);
                     saveSingleTaskDependencies(session, task, taskData);
-                    if (++counter % 50 == 0) {
+                    if (++counter % SCHEDULER_DB_NEW_TASKS_BATCH == 0) {
```

Comment:
Let's use `long` type for the counter?

Reply:
so we can go beyond 2 billions tasks in a workflow ?